app.py

```
@@ -575,19 +595,17 @@ def upload_form():
         elif kind.mime not in [
             "application/pdf",
             "application/msword",
             "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
             "text/plain",
         ]:
-            flash(
-                f"Unsupported file type: {original_filename}. Only PDF, DOC, DOCX, and TXT are allowed.",
-                "error",
-            )
-            continue
+            f"Unsupported file type: {original_filename}. Only PDF, DOC, DOCX, and TXT are allowed.",
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)